Tools/gdb/libpython.py

```diff
@@ -709,13 +709,18 @@ def write_repr(self, out, visited):
             pyop_value.write_repr(out, visited)
         out.write('}')
 
+    try:
+        _gdb_error = gdb.error
+    except AttributeError:
+        _gdb_error = RuntimeError
```

Comment:
This is currently at the PyDictObjectPtr class scope.  I can move this to the top of the class unless you think it may be useful elsewhere.